smithy-model/src/main/java/software/amazon/smithy/model/Model.java

```diff
@@ -761,6 +762,26 @@ public <T extends Shape> Stream<T> shapes(Class<T> shapeType) {
         return toSet(shapeType).stream();
     }
 
+    /**
+     * Gets a stream of shapes that have mixins.
+     *
+     * @return stream of shapes
+     */
+    public Stream<Shape> shapesWithMixins() {
```

Comment:
Given these are only used in one place and this class is already huge, I'd prefer to not add another utility method for this to Model.

Reply:
Resolved in commit: 40775d9